tools/clang/tools/dxil2spv/dxil2spvmain.cpp

```diff
@@ -48,22 +48,25 @@ int __cdecl wmain(int argc, const wchar_t **argv_) {
 #else
 int main(int argc, const char **argv_) {
 #endif // _WIN32
+  // Use same error return code for all dxil2spv failures.
+  const int ERROR_RETURN = 1;
```

Comment:
AFAIK it's idiomatic to use EXIT_SUCCESS and EXIT_FAILURE in `main`: https://en.cppreference.com/w/cpp/utility/program/EXIT_status